proposals/kubeflow-technical-oversight-committee.md

```diff
@@ -0,0 +1,184 @@
+
+  - Working together with Security WG to establish security guidelines to ensure Kubeflow follows
+    the CNCF security best practices.
+
+  - Advise the Conformance Committee on conformance rules and tests.
+
+- Project oversight and scope:
+
+  - Working together with Kubeflow Steering Committee (KSC) to establish guidelines for
+    Kubeflow distributions and third-party applications.
+
+  - Advise the KSC on creation and deletion Kubeflow working groups.
+
+  - Advise the KSC on accepting new Kubeflow components and archiving existing
+    Kubeflow components.
+
+  - Advise the KSC on changes for Kubeflow distributions and third-party applications.
+
+## Delegated Authority
+
+TOC may choose to delegate its authority to other committees as-needed.
+
+The committee currently recognizes this delegated authority for [To be set up]:
+
+- Kubeflow components guidance is delegated to the appropriate [Working Groups](../wgs/wg-governance.md).
+
+- Enforcing Kubeflow conformance program rules created by TOC and KSC is deleted to the\
+  [Conformance Committee](CONFORMANCE-COMMITTEE.md).
+
+## Committee Meetings
+
+TOC currently meets at least bi-weekly, or as-needed. Meetings are open to the public and held online,
+unless they pertain to sensitive or privileged matters. Examples of such matters are:
+
+- Private emails to the committee
+- Certain Escalations
+- Disputes between members
+- Security reports
+
+Meeting notes are available to members of the
+[kubeflow-discuss mailing](https://www.kubeflow.org/docs/about/community/#kubeflow-mailing-list)
+list, unless community member privacy requires otherwise.
+Public meetings will be recorded and the recordings made available publicly.
+
+Questions and proposals for changes to governance are posted as issues in the
+[kubeflow/community](https://github.com/kubeflow/community) repository, and the TOC invites community
+feedback there.
+
+## Committee members
+
+TOC is composed of 3 (three) members. They are elected according
+to the election policy [TODO: add link].
+Seats on the TOC are held by an individual, not by their employer.
+
+The current membership of the committee is (listed alphabetically by first name):
+
+<table>
+  <thead>
+    <tr>
+      <th><br>
+Member</th>
+      <th><br>
+Organization</th>
+      <th><br>
+Github Profile</th>
+      <th><br>
+Term Start</th>
+      <th><br>
+Term End</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td></td>
+      <td></td>
+      <td></td>
+      <td></td>
+      <td></td>
+    </tr>
+    <tr>
+      <td></td>
+      <td></td>
+      <td></td>
+      <td></td>
+      <td></td>
+    </tr>
+    <tr>
+      <td></td>
+      <td></td>
+      <td></td>
+      <td></td>
+      <td></td>
+    </tr>
+  </tbody>
+</table>
+
+## Emeritus Committee Members
+
+[This section will be populated when there are retired committee members.]
+
+## Decision process
+
+The TOC desires to always reach consensus.
+
```

Comment:
We should consider time limits for topics and a best practice for when a formal issue or PR must be created ?.   In order to drive for decisions, we should define a process for requesting a vote i.e.  can we take a vote before discussion, and at the 10 minute mark or other?   In addition, if a member objects to the vote results, what is the process to continue the conversation i.e. does the objecting member need to provide a plan to deliver more information before the discussion can continue.

Reply:
I agree with you @jbottum, any suggestions on how we can express it in the charter ?
For example, when making changes to the TOC charter we give at least 1 week community to provide comments: https://github.com/kubeflow/community/pull/701/files/5d5a3e473d464b9a036e527040e62adaaa86577c#diff-6a27463fa3d59d96139352472c4244cd45e5fd103c80fc00d5fe320b7c04ca76R183-R184

Reply:
Sounds good. I added it @jbottum